app/views/form_question/show.html.haml

```diff
@@ -3,24 +3,40 @@
 
 %div.container-fluid
   %h1 #{@form_type}
-  %p
-    %font{:color => "Crimson"} #{@error}
 
+  = form_tag form_user_path(:type=> @form_type) do
```

Comment:
Yeah the admin side needs this too, I was thinking about making it a partial and rendering them in the places we need them.